okapi-core/src/main/java/org/folio/okapi/service/impl/ModuleHandleFactory.java

```diff
@@ -12,6 +12,17 @@ private ModuleHandleFactory() {
     throw new IllegalAccessError("ModuleHandleFactory");
   }
 
+  /**
+   * Create module handle.
+   * @param vertx Vert.x handle
+   * @param desc launch descriptor
+   * @param id module ID
+   * @param ports ports to be available
+   * @param moduleHost module host override (for Docker)
+   * @param port port to be in use for module
+   * @param config configuration
+   * @return module handle
+   */
   public static ModuleHandle create(Vertx vertx, LaunchDescriptor desc, String id,
```

Comment:
In addition `id` should be renamed to `tenantId` to also have this information in the method signature. Some IDEs like Eclipse insert the method signature when doing auto-complete.

Reply:
Since the id is a module-Id using tenantId here would be very misleading.